docs/pages/server-access/guides/openssh.mdx

```diff
@@ -10,7 +10,13 @@ accept SSH certificates dynamically issued by a Teleport CA.
 
 Using Teleport and OpenSSH has the advantage of getting you up
 and running, but in the long run, we would recommend replacing `sshd` with `teleport`.
-We've outlined these reasons in [OpenSSH vs Teleport SSH for Servers?](https://goteleport.com/blog/openssh-vs-teleport/)
+`teleport` SSH servers have support for multiple features that are incompatible with OpenSSH:
+
+- [Automated host user creation](host-user-creation.mdx)
+- [Dynamic labels](../../management/labels.mdx)
+- [Session recording without SSH connection termination](recording-proxy-mode.mdx)
+- [Enhanced session recording](bpf-session-recording.mdx)
```

Comment:
Would an OpenSSH user know what this is?

Reply:
Updated with a better description